### PR TITLE
increase default retryable txn submission fee bump / other retryable tooling fixes

### PR DESCRIPTION
**Pull request by DZGoldman**
max base fee increase per window is (i.e., will be) 150%

### CODE REVIEW

**Review by fredlacs — COMMENTED** [review]
I think a slightly bigger refactor would make sense here. 
Should we make a utility function that handles the bump up?

**Comment by DZGoldman** [comment]
with the %150 increase per period cap, this should be pretty safe; what do you have in mind in particular?

**Comment by DZGoldman** [comment]
- added % increase utility func
- increased default % increase for until the 150% cap is enforced

**Comment by DZGoldman** [comment]
added some additional semi-related fixes that were surfaced here https://github.com/OffchainLabs/arbitrum/pull/1350

File: packages/arb-ts/src/lib/bridge_helpers.ts [hunk 1]
```diff
@@ -865,4 +865,8 @@ export class BridgeHelper {
     const whiteList = Whitelist__factory.connect(whiteListAddress, l1Provider)
     return whiteList.isAllowed(address)
   }
+
+  static percentIncrease(num: BigNumber, increase: BigNumber) {
```
Comment (fredlacs) [inline packages/arb-ts/src/lib/bridge_helpers.ts:869]: If we're doing a percent increase of 100%, wouldn't that be `num + num.mul(increase).div(100)`?

**Review by fredlacs — CHANGES_REQUESTED** [review]
One small naming thing that can be misleading, looks good other than that

File: packages/arb-ts/src/lib/bridge_helpers.ts (hunk 1 above)
Comment (DZGoldman) [reply packages/arb-ts/src/lib/bridge_helpers.ts:869]: lel true, it's poorly named. I'll change the logic tho (and keep the name)

**Review by fredlacs — APPROVED** [review]
LGTM

**Pull request merged**